torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -322,6 +322,10 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
   // 5. Check if it's a function attribute.
   if (const auto fnAttr = concreteType_->findFunctionAttribute(field)) {
     return std::make_shared<FunctionValue>(*fnAttr);
+  } else if (
+      const auto builtin_name = concreteType_->findBuiltinFunction(field)) {
+    return std::make_shared<BuiltinFunction>(
```

Comment:
we should push the conversion from a string -> Symbol earlier in the process (probably in `addBuiltinFunction`), just in case there are errors in the conversion.